setup.py

```diff
@@ -10,20 +10,25 @@ def get_version(filename):
     return re.search(r"__version__ = ['\"]([^'\"]+)['\"]", contents).group(1)
 
 
-version = get_version(os.path.join('django_perf_rec', '__init__.py'))
+def get_readme():
```

Comment:
Why are you changing all this?

Reply:
`with open()...` outside a function is a bit strange to me.

Reply:
It's not strange. Putting it into a function then directly calling that function has the same effect, but with an extra step to mask it. Please undo all these changes.